### DISCUSSION TITLE
good library to segment cells

### DISCUSSION THREAD

**Discussion by mariakesa** [quantumjot/btrack#247] [Ideas]
Hello:-)

Thank you for the wonderful software!

Do you have a suggestion for using a library for cell segmentation from images that would produce the csv data that would be compatible with btracker?

Thank you!

Maria

**Comment by quantumjot** [comment]
Hi @mariakesa! Thanks for your question. There are several really good open-source libraries for doing cell segmentation. What type of data do you have?

**Comment by mariakesa** [comment]
Hello! Thanks so much for your reply:-)

I have cellular phase contrast imaging data!

**Comment by quantumjot** [comment]
Seems like:
+ Cellpose https://github.com/MouseLand/cellpose or
+ Usiigaci (MaskRCNN) https://github.com/oist/Usiigaci

might be a good places to start. 

@KristinaUlicna - shall we add a notebook showing how to go from the segmentation to btrack?

**Comment by mariakesa** [comment]
Thank you so much! You know I worked with Carsen Stringer who is one of the makers of Cellpose:-)

**Comment by hftsai** [comment]
> Seems like:
> 
> * Cellpose https://github.com/MouseLand/cellpose or
> * Usiigaci (MaskRCNN) https://github.com/oist/Usiigaci
> 
> might be a good places to start.
> 
> @KristinaUlicna - shall we add a notebook showing how to go from the segmentation to btrack?

This is awesome. We are having a student to work on porting Usiigaci's result to work with bayesian tracker.
super impressive.

**Comment by quantumjot** [comment]
> This is awesome. We are having a student to work on porting Usiigaci's result to work with bayesian tracker.
super impressive.

Exciting! Please let us know if we can help. I'm really curious to know what the results are like.
